libs/editor/WordPressEditor/src/main/res/layout/fragment_gutenberg_editor.xml

```diff
@@ -1,87 +1,66 @@
 <?xml version="1.0" encoding="utf-8"?>
-
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout  xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:aztec="http://schemas.android.com/apk/res-auto"
```

Comment:
Nit: let's remove the unused namespace declaration, wdyt?

Reply:
👍 addressed in d16d1e9ee1